pkgs/top-level/all-packages.nix

```diff
@@ -24443,8 +24443,6 @@ with pkgs;
     buildPythonPackage buildPythonApplication setuptools pycsdr pydigiham;
   };
 
-  optparse-bash = callPackage ../development/libraries/optparse-bash { };
```

Comment:
Usually we populate aliases.nix with a throw and a suggestion; in the case I would pick `argbash`.

Reply:
Addressed.